compiler/rustc_resolve/src/diagnostics.rs

```diff
@@ -1163,8 +1163,6 @@ impl<'a, 'tcx> Resolver<'a, 'tcx> {
             Some(x) => Some(x),
         } {
             let in_module_is_extern = !in_module.def_id().is_local();
-            // We have to visit module children in deterministic order to avoid
-            // instabilities in reported imports (#43552).
```

Comment:
This comment was added in #43552 but is no longer relevant. It should've been removed in #65043. Doing it now instead :)